cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperatorConfig.java

```diff
@@ -192,6 +198,8 @@ public static ClusterOperatorConfig fromMap(Map<String, String> map, KafkaVersio
         RbacScope rbacScope = parseRbacScope(map.get(STRIMZI_RBAC_SCOPE));
         Labels customResourceSelector = parseLabels(map, STRIMZI_CUSTOM_RESOURCE_SELECTOR);
         String featureGates = map.getOrDefault(STRIMZI_FEATURE_GATES, "");
+        String operatorInstance = map.get(CLUSTER_OPERATOR_HOST);
```

Comment:
Is it required? Should we check here if it is set or not?

Reply:
A controller instance is required for v1 events, so I'll ensure it's injected.